### PR TITLE
add supervisorLockedError

### PR DESCRIPTION
**Pull request by MoranF**
Connects-To: #20

Change-Type: patch

### CODE REVIEW

File: lib/errors.ts [hunk 1]
```diff
@@ -64,6 +64,24 @@ export class ResinMalformedToken extends ResinError {
...
+ * @class
+ * @public
+ *
+ * @param {String} token - token
+ * @return {Error} error instance
+ *
+ * @example
+ * throw new errors.SupervisorLockedError()
+ */
+export class SupervisorLockedError extends ResinError {
```
Comment (LucianBuzzo) [inline lib/errors.ts]: Can you name this class with `Resin` prefix to match the other errors, please?
`ResinSupervisorLockedError`

**Comment by LucianBuzzo** [comment]
@MoranF Can you fix the typo in this commit message please?

File: lib/errors.ts (hunk 1 above)
Comment (MoranF) [reply lib/errors.ts]: So do I need to call ResinSupervisorLockedError also from the sdk where using this error?

Comment (LucianBuzzo) [reply lib/errors.ts]: Yep, We'll use `ResinSupervisorLockedError` instead of `SupervisorLockedError`.

**Pull request merged**